### PR TITLE
update batch sample to use v3 API

### PR DESCRIPTION
**Pull request by chlandsi**
## Purpose

update batch sample to use v3 API.

## Pull Request Type

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Documentation content changes
- [ ] Other

### CODE REVIEW

File: samples/batch/python/README.md [hunk 1]
```diff
@@ -8,25 +8,13 @@ Follow these steps for the installation:
 
 1. Go to https://editor.swagger.io.
 1. Click **File**, then click **Import URL**.
-1. Enter the Swagger URL including the region for your Speech Services subscription: `https://<your-region>.cris.ai/docs/v2.0/swagger`.
+1. Enter the Swagger URL for the Speech Services API: `https://westus.dev.cognitive.microsoft.com/docs/services/speech-to-text-api-v3-0/export?DocumentFormat=Swagger&ApiName=Speech%20to%20Text%20API%20v3.0`.
```
Comment (zhouwangzw) [inline samples/batch/python/README.md:11]: is the "dev" in "westus.dev.xxxx" expected?

Comment (zhouwangzw) [inline samples/batch/python/README.md:11]: Shall we mention V3 api somewhere?

Comment (chlandsi) [reply samples/batch/python/README.md:11]: Yes, that's the one we link to also from https://docs.microsoft.com/en-gb/azure/cognitive-services/speech-service/batch-transcription.

File: samples/batch/python/python-client/main.py [hunk 2]
```diff
@@ -25,118 +23,180 @@
...
+# Set model information when doing transcription with custom models
+MODEL_REFERENCE = None  # guid of a custom model
+
+
+def transcribe_from_single_blob(uri, properties):
+    """
+    Transcribe a single audio file located at `uri` using the settings specified in `properties`
+    using the base model for the specified locale.
+    """
+    transcription_definition = cris_client.Transcription(
```
Comment (zhouwangzw) [inline samples/batch/python/python-client/main.py:38]: is it "transcription definition" or already "transcription response"?

File: samples/batch/python/README.md (hunk 1 above)
Comment (chlandsi) [reply samples/batch/python/README.md:11]: Not sure, we don't really version the samples. The C# sample doesn't mention it in the description

File: samples/batch/python/python-client/main.py (hunk 2 above)
Comment (chlandsi) [reply samples/batch/python/python-client/main.py:38]: Here, it is really the definition (hasn't been sent yet)

**Pull request merged**